dbt_artifacts_ext/converter/__init__.py

```diff
@@ -50,13 +50,17 @@ def __init__(self):
         log.debug(f"Output directory: {self.output_dir}")
 
         resource_types = os.getenv("DBT_ARTIFACTS_RESOURCE_TYPES")
+        exclude_package_names = os.getenv("DBT_ARTIFACTS_EXCLUDE_PACKAGES")
```

Comment:
Let's rename this variable (and all references to it) to align with the setting name.

```suggestion
        exclude_packages = os.getenv("DBT_ARTIFACTS_EXCLUDE_PACKAGES")
```